monitoring/publishing/db_update_sqlite.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+"""
+`db_update_sqlite.py` - Syncs data from external database into local SQLite DB.
+                      -  It will be run as a standalone operation via cron.
+"""
+import configparser
+import logging
+import os
+import sys
+
+import django
+from django.db import DatabaseError
+import pandas as pd
+
+
+BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
+
+# Find the root and the Django project
+sys.path.append(BASE_DIR)
+
+# Set up Django settings to run this `db_update_sqlite.py` as standalone file
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "monitoring.settings")
+
+# Initialize and setup Django
+django.setup()
```

Comment:
Can you move the the `import django` in just before the `django.setup()` and then encapsulate all these lines,  in an `if __name__ == "__main__":`  please?

```suggestion
if __name__ == "__main__":
    # Set up to run this `db_update_sqlite.py` script as a standalone file
    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "monitoring.settings")
    import django
    django.setup()
```

Reply:
We CAN'T do that here, Adrian. The script won't run. 

We always get Traceback (most recent call last):
  File "monitoring/publishing/db_update_sqlite.py", line 24, in <module>
    from monitoring.publishing.models import (
  File "/usr/share/DJANGO_MONITORING_APP/monitoring/publishing/models.py", line 7, in <module>

Reply:
Ok, is that just an ordering issue then? Could the import got before the `os.environ` or is there another line that depends on this?

Reply:
Also, as I said on Slack, if the` if...main` bit wouldn't t work, can you at least group the lines I've suggested together (just without the if encapsulation)?

Reply:
I have removed  if encapsulation. But, Still I cant alter the order of the it.

Reply:
Sorry, I meant the _suggested_ encapsulation, not the `if __name__ == "__main__":` at the bottom of the file around line 213. I'll revert that bit of the change and get this merged.

Reply:
I've put the if __name__ back in at the end of the file.